lib/ExampleDriver.js

```diff
@@ -0,0 +1,37 @@
+/*!
+ * Copyright (c) 2020 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+/**
+ * Example key suite driver for use with CryptoLd. When adding support for
```

Comment:
Instructions for developers implementing a driver library for use with `crypto-ld`.